src/app.rs

```diff
@@ -3,7 +3,7 @@ use crate::{
     ui::popup::{info_popup::InfoPopUp, PopUp},
 };
 use ansi_to_tui::IntoText;
-use color_eyre::eyre::bail;
+use color_eyre::eyre::{bail};
```

Comment:
Unecessary braces. They should be used when importing different parts in the same package.
```suggestion
use color_eyre::eyre::bail;
```